FirebaseRemoteConfig/Sources/RCNConfigRealtime.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2019 Google
```

Comment:
New files should be `Copyright 2022 Google LLC`

Reply:
Updated the copyright and is there an alternative that would be preferred over typedef? I don't think I'll be adding any other typedefs to the public API.

Reply:
Use the actual types instead of the typedef - See http://goshdarnblocksyntax.com/. 

Or declare the typedef somewhere else besides a public header

Reply:
Moved the typedef to a private header file and fixed CLA problem